tests/automation.js

```diff
@@ -295,6 +295,16 @@ casper.test.begin("unit tests", 39 + gfxTests.length, function(test) {
     });
 
     casper
+    .thenOpen("http://localhost:8000/index.html?midletClassName=tests/drag/Coalesce&jad=tests/midlets/drag/coalesce.jad&jars=tests/tests.jar&logConsole=web,page&logLevel=log")
```

Comment:
Nit: this only tests pen events, we could test gesture events as well (they should work in the same way, so it's less important)